<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSelectorClass</class>
 <widget class="QWidget" name="ParameterSelectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParameterSelector</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources/main.qrc">
       <normaloff>:/new/prefix1/refresh.png</normaloff>:/new/prefix1/refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QCheckBox" name="autoBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ParameterSelectorClass</receiver>
   <slot>currentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>currentIndexChanged(int)</slot>
 </slots>
</ui>
